src/main/kotlin/no/nav/familie/ef/sak/vedtak/domain/Vedtak.kt

```diff
@@ -217,7 +217,7 @@ data class InntektWrapper(
 )
 
 data class TilleggsstønadWrapper(
-    val harTilleggsstønad: Boolean,
+    val harTilleggsstønad: Boolean?,
```

Comment:
Fint med en kommentar her om at dette er for historiske vedtak, og at denne alltid er null for nyere vedtak..?